src/operon.ts

```diff
@@ -231,6 +231,17 @@ export class Operon {
     }
     void this.recoverPendingWorkflows();
     this.initialized = true;
+
+    for ( const v of this.registeredOperations) {
```

Comment:
My understanding is we can have multiple init methods and they may be applied in whatever order?

Reply:
Yes . that is the current behavior. Perhaps in the future we can add a param for order

Reply:
> Yes . that is the current behavior. Perhaps in the future we can add a param for order

Or... you could do the steps in order in 1 function :-D